mm/2s2h/Rando/ActorBehavior/ObjSnowball.cpp

```diff
@@ -0,0 +1,201 @@
+    { { -1185.00f, 630.00f }, RC_SNOWHEAD_TEMPLE_ICICLE_ROOM_SMALL_SNOWBALL_03 },
+    { { -810.00f, 540.00f }, RC_SNOWHEAD_TEMPLE_ICICLE_ROOM_SMALL_SNOWBALL_04 },
+    { { -870.00f, 510.00f }, RC_SNOWHEAD_TEMPLE_ICICLE_ROOM_SMALL_SNOWBALL_05 },
+    { { -840.00f, 60.00f }, RC_SNOWHEAD_TEMPLE_SNOW_ROOM_SMALL_SNOWBALL_01 },
+    { { -885.00f, 75.00f }, RC_SNOWHEAD_TEMPLE_SNOW_ROOM_SMALL_SNOWBALL_02 },
+    { { -810.00f, 90.00f }, RC_SNOWHEAD_TEMPLE_SNOW_ROOM_SMALL_SNOWBALL_03 },
+    { { -855.00f, 120.00f }, RC_SNOWHEAD_TEMPLE_SNOW_ROOM_SMALL_SNOWBALL_04 },
+    { { -870.00f, 165.00f }, RC_SNOWHEAD_TEMPLE_SNOW_ROOM_SMALL_SNOWBALL_05 },
+    { { -1290.00f, -615.00f }, RC_SNOWHEAD_TEMPLE_SNOW_ROOM_SMALL_SNOWBALL_06 },
+    { { -1335.00f, -585.00f }, RC_SNOWHEAD_TEMPLE_SNOW_ROOM_SMALL_SNOWBALL_07 },
+    { { -1305.00f, -570.00f }, RC_SNOWHEAD_TEMPLE_SNOW_ROOM_SMALL_SNOWBALL_08 },
+    { { 427.00f, -613.00f }, RC_TWIN_ISLANDS_LARGE_SNOWBALL_01 },
+    { { 65.00f, 345.00f }, RC_TWIN_ISLANDS_LARGE_SNOWBALL_02 },
+    { { -179.00f, -876.00f }, RC_TWIN_ISLANDS_LARGE_SNOWBALL_03 },
+    { { 851.00f, 643.00f }, RC_TWIN_ISLANDS_LARGE_SNOWBALL_04 },
+    { { 371.00f, 778.00f }, RC_TWIN_ISLANDS_LARGE_SNOWBALL_05 },
+    { { -563.00f, -523.00f }, RC_TWIN_ISLANDS_LARGE_SNOWBALL_06 },
+    { { 1203.00f, 109.00f }, RC_TWIN_ISLANDS_LARGE_SNOWBALL_07 },
+    { { -1029.00f, -25.00f }, RC_TWIN_ISLANDS_LARGE_SNOWBALL_08 },
+    { { -880.00f, 497.00f }, RC_TWIN_ISLANDS_LARGE_SNOWBALL_09 },
+    { { -395.00f, 677.00f }, RC_TWIN_ISLANDS_LARGE_SNOWBALL_10 },
+    { { 5.00f, -560.00f }, RC_TWIN_ISLANDS_LARGE_SNOWBALL_11 },
+    { { 60.00f, -46.00f }, RC_TWIN_ISLANDS_LARGE_SNOWBALL_12 },
+    { { -1525.00f, 1264.00f }, RC_TWIN_ISLANDS_SMALL_SNOWBALL_01 },
+    { { -1343.00f, 776.00f }, RC_TWIN_ISLANDS_SMALL_SNOWBALL_02 },
+    { { 99.00f, -221.00f }, RC_TWIN_ISLANDS_SMALL_SNOWBALL_03 },
+    { { -1349.00f, 114.00f }, RC_TWIN_ISLANDS_SMALL_SNOWBALL_04 },
+    { { -1264.00f, 161.00f }, RC_TWIN_ISLANDS_SMALL_SNOWBALL_05 },
+    { { -1326.00f, 172.00f }, RC_TWIN_ISLANDS_SMALL_SNOWBALL_06 },
+
+};
+
+RandoCheckId IdentifySnowball(Vec3f pos) {
+    RandoCheckId randoCheckId = RC_UNKNOWN;
+
+    auto it = snowballMap.find({ pos.x, pos.z });
+    if (it == snowballMap.end()) {
+        return randoCheckId;
+    }
+
+    return it->second;
+}
+
+void SpawnSnowballDrop(Vec3f pos, RandoCheckId randoCheckId) {
+    CustomItem::Spawn(
+        pos.x, pos.y, pos.z, 0, CustomItem::KILL_ON_TOUCH | CustomItem::TOSS_ON_SPAWN | CustomItem::ABLE_TO_ZORA_RANG,
+        randoCheckId,
+        [](Actor* actor, PlayState* play) {
+            RandoSaveCheck& randoSaveCheck = RANDO_SAVE_CHECKS[CUSTOM_ITEM_PARAM];
+            randoSaveCheck.eligible = true;
+        },
+        [](Actor* actor, PlayState* play) {
+            auto& randoSaveCheck = RANDO_SAVE_CHECKS[CUSTOM_ITEM_PARAM];
+            RandoItemId randoItemId = Rando::ConvertItem(randoSaveCheck.randoItemId);
```

Comment:
```suggestion
            RandoItemId randoItemId = Rando::ConvertItem(randoSaveCheck.randoItemId, CUSTOM_ITEM_PARAM);
```

Reply:
wait, actually, you're doing this below, and not even using this variable, just remove this
```suggestion
```